src/languages/en.ts

```diff
@@ -2107,6 +2107,7 @@ export default {
         tripSummary: 'Trip summary',
         departs: 'Departs',
         errorMessage: 'Something went wrong. Please try again later.',
+        phoneError: 'To book travel, your primary login must be a valid email',
```

Comment:
Could we link them in the error to the place where they can update this?

Reply:
Maybe! Giving it a shot.
I'm also considering rephrasing this, as we call it "Default contact method" in newdot, istead of "primary login" like we do in oldDot. What do you think?

Reply:
Ok, it `can` be done, but we don't really have an existing structure for it. We could do something similar to what we do for receipt errors in `DotIndicatorMessage`, but it would be, basically, writing another exception for this specific error. Since we have other similar errors that don't really link anywhere, I don't think it is worth it for this cse.

Thoughts?

Reply:
I think it would make for a better, more useful error message if we did. Perhaps we have a contributor do it as follow-up?

Reply:
What do you think about this @Gonals?